app/models/card.rb

```diff
@@ -4,18 +4,24 @@ class Card < ActiveRecord::Base
   belongs_to :user
   validates :original_text, :translated_text, :review_date, :user_id, presence: true, on: [:create, :update]
   validate :text_are_not_equal
-  before_create :set_default_review_date
-  scope :for_review, -> { where('review_date <= ?', DateTime.now).order('random()') }
+  scope :for_review, -> { where('review_date <= ?', DateTime.current).order('random()') }
```

Comment:
Line is too long. [89/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.